azurerm/internal/services/containers/kubernetes_addons.go

```diff
@@ -88,6 +88,7 @@ func schemaKubernetesAddOnProfiles() *schema.Schema {
 					Type:     schema.TypeList,
 					MaxItems: 1,
 					Optional: true,
+					Computed: true,
```

Comment:
we need to revert this - Azure Public is the only Environment where this is enabled, and that's being EOL in the near future

Reply:
from the http response, it seems the value is default `false`. I think there is no harm after adding `Computed: true`?

Reply:
that's only true for certain Kubernetes versions in certain Azure Environments - this cannot be specified in other Environments, so there's an explicit difference between `false` and `null` - unfortunately this is the design of the API.

Reply:
Got it. has reverted